### DISCUSSION TITLE
Manifestation Statements

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#463] [Q&A] [answered]
What's the deal with manifestation statements? Are they being used at all? What is the distinction between http://rdaregistry.info/Elements/m/P30107 and http://rdaregistry.info/Elements/m/P30284 in practice that would necessitate a new element for manifestation edition statement? How do we plan to use [MARC 881](https://www.loc.gov/marc/bibliographic/bd881.html)? Most importantly, does any of this affect our work?

**Comment by GordonDunsire** [comment, marked as answer]
The RDA manifestation statement elements, which are distinguished by having a label that begins with 'manifestation' and ends with 'statement', were added to official RDA in the 3R project; they are derived from an LRM element that was not present in FRBR, FRAD, or FRSAD. The value of a manifestation statement element is an unstructured description that is transcribed from the manifestation that is being described. No other source of information is used. The transcription should follow a set of transcription rules chosen by the cataloguing agency; these govern omissions, punctuation, capitalization, etc.. Field 881 should map directly to these RDA elements.

The other statements elements are legacy elements that are based on ISBD areas. They are usually structured descriptions because they are assembled from the values of other elements which may not only be found in the manifestation being described, but also in external reference sources, etc. Those values may include corrections and information not present in the manifestation. These statement elements may be recorded with unstructured descriptions that are very similar to those of manifestation statement elements, but RDA gives a general option to use the corresponding manifestation statement element rather than the statement element. The intention is to soft-deprecate the statement elements in due course; they are legacy elements that are retained to bridge between ISBD/AACR2 and ISBD/LRM. Note that ISBDM does not retain any statement elements or areas, and accommodates only manifestation statement elements.

We have agreed that the transform cannot detect if the MARC 21 fields for statement elements are unstructured descriptions that can be mapped to RDA manifestation statement elements. Instead, they are always mapped to the legacy RDA statement elements.

In short, field 881 only maps to manifestation statement elements, and other statement fields only map to legacy statement elements.
